### DISCUSSION TITLE
Which variable is surface roughness for V2 surface files

### DISCUSSION THREAD

**Discussion by uturuncoglu** [ufs-community/ufs-weather-model#2418] [Q&A] [answered]
I am trying to use V2 surface files (activated by `export V2_SFC_FILE=true` in the RT script) for land coupling and I realized that the surface files have two variables for the surface roughness and those are 

```
	double zorli(Time, yaxis_1, xaxis_1) ;
		zorli:long_name = "zorli" ;
		zorli:units = "none" ;
		zorli:coordinates = "geolon geolat" ;
		zorli:missing_value = -1.e+20 ;
	double zorlw(Time, yaxis_1, xaxis_1) ;
		zorlw:long_name = "zorlw" ;
		zorlw:units = "none" ;
		zorlw:coordinates = "geolon geolat" ;
		zorlw:missing_value = -1.e+20 ;
```

It seems that one of these is for ice and other is for water and there is no land component. In original files, I was using `zorl` variable but I am not sure which variable in v2 file corresponds to `zorl`. Any suggestion would be helpful.

**Comment by uturuncoglu** [comment]
@barlage @junwang-noaa do you have any idea?

**Comment by jkbk2004** [comment]
@GeorgeGayno-NOAA Any comment about zorl/zorli/zorlw with the v2 surface file option?

**Reply by GeorgeGayno-NOAA** [reply, marked as answer]
The v2 surface file contains zorli (roughness at points with some ice) and zorlw (roughness at points with some open water). There is no record for roughness at land because the land model determines that field from vegetation type via a look-up table.

**Comment by gspetro-NOAA** [comment]
@uturuncoglu Did you need any further assistance? If George Gayno's response answered your question, we can "Mark as answer." Otherwise, let us know what further information you need. :)

**Reply by uturuncoglu** [reply]
@gspetro-NOAA Yes but this means that we could not use V2 files. At this point, I am trying to make the component model work with original files.
